docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -154,4 +154,54 @@ has the following advantages:
 
 If you want to search all data in your S3 bucket for new samples
 instead of only newly added data,
-then set `datasource.process_all=True` in your docker run command.
\ No newline at end of file
+then set `datasource.process_all=True` in your docker run command.
+
+
+Network traffic
+---------------
+
+Please ensure that the region of your bucket and where you intend to be running the
+Lightly Docker instance are be the same (e.g. `eu-central-1`). If the region is not
+the same there can be `degraded transfer speeds and additional costs will be incurred by AWS <https://aws.amazon.com/premiumsupport/knowledge-center/s3-transfer-data-bucket-instance/>`_!
+This also applies to other cloud providers. We highly recommend using the same region.
+
+
+The worker causes significant network traffic at the following steps:
+
+For image datasets:
+
+- The corruptness check downloads the complete dataset.
+- Training the embedding model downloads the complete dataset once each epoch.
+- Embedding and pretagging each download the non-corrupt dataset.
+- Dumping the selected dataset downloads it.
+- Updating the selected dataset in the Lightly platform
+  will first download all newly selected images to compute their metadata.
+  If you configured this step to upload also thumbnails,
+  it will upload one thumbnail for each newly selected image.
+- Creating the report with images overlayed in the scatterplot will download about 50 images.
+- Uploading the report needs a few MB.
+
+
+For video datasets:
+
+Depending on the video format, downloading a full video might be necessary to download a single frame.
+Thus downloading x frames from y different videos might download all y videos in worst case.
```

Comment:
I'd personally go for upper case variables in a text block like this. 
x --> X
y --> Y
...